doc/api/buffer.md

```diff
@@ -2227,6 +2366,9 @@ console.log(buf);
 <!-- YAML
 added: v0.5.5
 changes:
+  - version: v11.0.0-pre
```

Comment:
We use `REPLACEME` as a placeholder for the version in which a PR lands, it’s picked up automatically by our tooling that way :)

Reply:
Ah; understood.